.github/workflows/build.yml

```diff
@@ -17,12 +17,27 @@ jobs:
       - name: Run formatters
         id: run_formatters
         run: |
-          ./.githooks/pre-commit
+          ./.githooks/check-format
+        shell: bash
+  
+  check_docs:
```

Comment:
Why do we separate check-docs and check-format in two jobs?

Reply:
That's better because they could run in parallel and provide results independently.